packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -341,6 +345,12 @@ describe("SharedTree benchmarks", () => {
 
 	describe("acking local commits", () => {
 		const localCommitSize = [1, 25, 100, 500, 1000];
+		const builder = new SchemaBuilderBase(FieldKinds.optional, {
+			scope: "test",
+			libraries: [leaf.library],
+		});
+		const schemaGeneralized = builder.intoSchema(Any);
```

Comment:
So before this change, was there no schema available for sharedtree(  including a generalized one)? Was this generalized schema  (as opposed to flexfield/rootField) recently introduced?

Reply:
To be honest I'm not entirely clear how it works :) just followed an existing pattern in other tests that used compressed encoding. I'm hoping the SharedTree team will point out if there's anything weird with the usage pattern.